lib/core/src/display/shape.rs

```diff
@@ -1,5 +1,6 @@
 //! This is a root module for shapes, 2-dimensional graphical elements.
 
 pub mod compound;
+pub mod glyph;
 pub mod primitive;
-pub mod text;
+//pub mod text;
```

Comment:
kahem!

Reply:
But soon I'll restore this module as a refactored text field. I'll just add a comment.